modules/services/getmail.nix

```
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.getmail;
+
+  accounts = filter (a: a.getmail.enable)
+    (attrValues config.accounts.email.accounts);
+
+  renderConfigFilepath = a: ".getmail/getmail${a.name}";
```

Could include `$HOME` here:
```suggestion
  renderConfigFilepath = a: "\${HOME}/.getmail/getmail${a.name}";
```

---

Yes that's better.